src/NHSD.BuyingCatalogue.Solutions.Application/Commands/UpdateSolutionContactDetails/UpdateSolutionContactDetailsExecutor.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using NHSD.BuyingCatalogue.Solutions.Application.Commands.Execution;
 using NHSD.BuyingCatalogue.Solutions.Application.Persistence;
+using NHSD.BuyingCatalogue.Solutions.Application.Queries.GetContactDetailBySolutionId;
 using NHSD.BuyingCatalogue.Solutions.Application.Queries.GetSolutionById;
```

Comment:
Is this using statement still referenced

Reply:
👍